.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Please add that in your global ignore file or in .git/info/exclude and remove it from the pr.